src/main/java/seedu/address/model/exam/ExamDate.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.model.exam;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+
+
+/**
+ * ExamDate class represents the date of the exam.
+ */
+public class ExamDate {
+    public static final String DATE_CONSTRAINTS =
+            "Exam Date should be in YYYY-MM-DD format and must be a valid date.";
```

Comment:
Just want to clarify if it would be better if it were in the form of dd-MM-yyyy vs yyyy-MM-dd

Reply:
Okay I'll standardized it to be dd-mm-yyyy just like the date format in the deadline class.